src/views/Books/Books.js

```diff
@@ -0,0 +1,29 @@
+import React, { useState, useEffect } from 'react';
+
+import { fetchBooks } from '../../services/books';
+
+
+
+export default function Books() {
+  const [books, setBooks] = useState([]);
+  useEffect(()=> {
+    const everyBookFunct = async () => {
+      const everyBook = await fetchBooks();
+      setBooks(everyBook);
+    };
+    
+    everyBookFunct();
+  }, []);
+
+  return (
+    <div>Novels:
+      {books.map((data)=> (
+        <div className='books' key={data.id}>
+          <h1>{data.title}</h1>
+        </div>
```

Comment:
Great job mapping over the books to display them on the page. We can also consider making this a presentational component to follow the separation of concerns, although it is not required for this lab.